pythran/pythonic/numpy/fft/c2c.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef PYTHONIC_NUMPY_FFT_C2C_HPP
+#define PYTHONIC_NUMPY_FFT_C2C_HPP
+
+#include "pythonic/include/numpy/fft/c2c.hpp"
+#include "pythonic/utils/functor.hpp"
+#include "pythonic/include/utils/array_helper.hpp"
+#include "pythonic/types/ndarray.hpp"
+#include "pythonic/builtins/None.hpp"
+#include "pythonic/numpy/concatenate.hpp"
+#include "pythonic/numpy/zeros.hpp"
+
+
+#include <array>
+#include <cstring>
+#include <cmath>
+
+#include "pythonic/numpy/fft/pocketfft.hpp"
+
+PYTHONIC_NS_BEGIN
+
+namespace numpy
+{
+  namespace fft
+  {
+    using pocketfft::stride_t;
+    using pocketfft::shape_t;
+    using ldbl_t = typename std::conditional<sizeof(long double)==sizeof(double), double, long double>::type;
+
+    template<typename T> T norm_fct(int inorm, size_t N)
+    {
+        if (inorm==0) return T(1);
+        if (inorm==2) return T(1/ldbl_t(N));
+        if (inorm==1) return T(1/sqrt(ldbl_t(N)));
+            throw std::invalid_argument("invalid value for inorm (must be 0, 1, or 2)");
```

Comment:
What kind of error is raised in Python? you should use the same here, e.g. `types::ValueError`

Reply:
using enum class now so removed the exception